bravado/client.py

```diff
@@ -215,6 +215,16 @@ def construct_params(self, request, op_kwargs):
             if not remaining_param.required and remaining_param.has_default():
                 marshal_param(remaining_param, None, request)
 
+    def log_warning_for_deprecated_op(self):
+        # TODO: Move this logic to bravado_decorators
```

Comment:
As the fields are only used for displaying the warning message, i was reluctant to add all the if's to check the `None`ness of the fields and not displaying that field accordingly. I feel `x-foo: None` in the message conveys that it is not provided.

I would also vote to move this method to some other module, as it is not super related to `client`.